creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -490,13 +550,17 @@ const _issueUpdateReplicaSetOp = async (
       )}`
     )
 
-    if (!issueReconfig) return response
+    if (!issueReconfig) {
+      response.error = 'success_issue_reconfig_disabled'
```

Comment:
this isn't an error?

Reply:
That's just the name of the variable. I was going to rename it but having the error be caused by reconfig being disabled needed reasonable.

Reply:
but then the job would be marked as an error right? even though it did not actually "error"

Reply:
The resulting metric is called `result`, it's just called `error` in that function

Reply:
ah yeah good point